Cargo.toml

```diff
@@ -56,11 +56,13 @@ hpke-rs = "0.2.0"
 hpke-rs-crypto = "0.2.0"
 hpke-rs-rust-crypto = "0.2.0"
 http = "1"
+js-sys = "=0.3.66"
```

Comment:
Why add this dependency? And why pin the version?

Reply:
FWIW, pinning versions can downgrade other dependencies, which could explain the bug you're seeing when you run the tests locally.

Reply:
This was a (vain) attempt to get the tests working again, as the bug appears more often in the newer version of workers.